content/en/tracing/guide/add_span_md_and_graph_it.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Add span metadata and graph it using Trace Analytics
+kind: guide
+disable_toc: true
+further_reading:
+- link: "tracing/setup/"
+  tag: "Documentation"
+  text: "Learn how to setup APM tracing with your application"
+- link: "tracing/visualization/services_list/"
+  tag: "Documentation"
+  text: "Discover the list of services reporting to Datadog"
+- link: "tracing/visualization/resource"
+  tag: "Documentation"
+  text: "Dive into your resource performance and traces"
+- link: "tracing/visualization/trace"
+  tag: "Documentation"
+  text: "Understand how to read a Datadog Trace"
+---
+
+{{< img src="tracing/guide/add_span_md_and_graph_it/span_md_6.gif" alt="comparaison 2" responsive="true" style="width:90%;">}}
```

Comment:
The gif here would be clearer with a top list - the graph as is doesn't necessarily show immediately why I want it. I'd even go further and make it a top list - and export it to a dashboard. The dashboard/monitor use case should definitely be covered here.

Reply:
How's the new gif?